tests/model_selection/dask_searchcv/test_model_selection.py

```diff
@@ -1031,7 +1031,7 @@ def train(y, norm1_wt, needs_proba):
     gscv = dcv.GridSearchCV(
         estimator,
         estimator_params,  # parameters
-        scoring=make_scorer(metric, greater_is_better, needs_proba),
+        scoring=make_scorer(score_func=metric, greater_is_better=greater_is_better, needs_proba=needs_proba),
```

Comment:
Just checked. This should be backward compatible in sklearn 0.23.x

https://github.com/scikit-learn/scikit-learn/blob/0.23.1/sklearn/metrics/_scorer.py#L534